lib/sdk/sdk.dart

```diff
@@ -7,6 +7,7 @@ library atom.dartino.sdk;
 import 'dart:async';
 
 import 'package:atom/atom.dart';
+import 'package:atom/node/notification.dart';
```

Comment:
Yes, but much of this SDK code will be removed after I've validated that the new dartino launch code covers all the necessary bases.